src/vec3.rs

```diff
@@ -417,7 +417,7 @@ impl_vecn_scalar_bit_op_traits!(UVec3, u32, XYZU32);
 impl_vecn_bit_op_traits!(UVec3, XYZU32);
 
 #[test]
-fn test_vec3_private() {
+fn test_vec3_mul_add() {
```

Comment:
Now that this is public, is would be good to move these to `tests/vec*.rs`. Not mandatory for this PR though, only if you feel like it. I can tidy it up later otherwise.

Reply:
Please do. I'm not sure how to port this over with the macro heavy tests there.